src/dnx/TraceWriter.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+#include "xplat.h"
+
+class TraceWriter
```

Comment:
Hard to tell. I am a big fan of `snake_case` in C/C++ but these files are using hungarian + camelCasing + Windows. I don't think spending time on just changing casing is really worth it, so I wanted to make it kind of similar to what we have. Note that this is the only class in these files so far so as far as class names go it can't be inconsistent.